constructr-machine/src/main/scala/de/heikoseeberger/constructr/machine/ConstructrMachine.scala

```diff
@@ -147,7 +147,11 @@ final class ConstructrMachine[A: Coordination.AddressSerialization, B <: Coordin
   }
 
   when(State.AddingSelf, coordinationTimeout) {
-    case Event(Coordination.SelfAdded(_), _) => goto(State.RefreshScheduled).using(stateData.copy(coordinationRetriesLeft = coordinationRetries))
+    case Event(Coordination.SelfAdded(context: B#Context), _) =>
```

Comment:
@hseeberger, this is an unchecked type pattern. We're facing a compile-time warning because of that:
```
Warning:(150, 50) abstract type pattern B#Context is unchecked since it is eliminated by erasure
    case Event(Coordination.SelfAdded(context: B#Context), _) =>
```
How do you prefer to deal with it?